pkg/webhook/webhook.go

```diff
@@ -74,3 +75,7 @@ func cipherSuites() []string {
 	}
 	return strings.Split(cipherSuitesEnv, ",")
 }
+
+func tlsMinVersion() string {
```

Comment:
When os.Getenv("TLS_MIN_VERSION") is empty, the server will default to "1.0", maybe worth a comment, maybe not.

Reply:
Adding comment